<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteMemberDialog</class>
 <widget class="QDialog" name="DeleteMemberDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Member</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	font: 13pt &quot;Arial Rounded MT Bold&quot;;
background-color:rgba(1,25,53,200);
color:rgba(208,233,255);
border-color:rgba(1,25,53,90);
border-radius:3px;
}
QLabel
{
font: 11pt &quot;Arial Rounded MT Bold&quot;;
color:rgb(1, 25, 53);
}
QLineEdit
{
font: 11pt &quot;Arial Rounded MT Bold&quot;;
           border-radius:3px;
         background-color:rgba(208,233,255);
      color:rgba(1,25,53);
}
QRadioButton
{
font: 11pt &quot;Arial Rounded MT Bold&quot;;
color:rgb(1, 25, 53);
}</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="vipButton">
      <property name="text">
       <string>VIP</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="svipButton">
      <property name="text">
       <string>Super VIP</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>301</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Birthday</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="yearEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="monthEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="dayEdit"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Phone No.</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="pointLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="phoneNoEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="searchButton">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Points</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Total Points</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="totalPointLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>320</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateButton">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>horizontalLayoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
